frontend/src/components/util/routeInjector.tsx

```diff
@@ -40,7 +43,7 @@ export default function RouteInjector({ children }: any) {
             : null,
         getAllEditionsFromPage
     );
-    const latestEdition = availableEditions?.at(0);
+    const latestEdition = availableEditions?.at(availableEditions?.length - 1);
```

Comment:
This is needed? (Just making sure it's not a residu of when we where both editing these files)

Reply:
This is because editions get added to a list so the newest edition is the last one in the list.